### PR TITLE
Add timezone to time field

### PR DESCRIPTION
**Pull request by kurtw**
### What this does
This primarily adds a timezone to the time field. Simple enough. This also changes a bit of the formatting in which to show the dates.

#### Testing
`./vendor/bin/phpunit` from the cms-toolkit base directory.

#### Review
@willbarton 
@dpford

### CODE REVIEW

File: inc/meta-box-callbacks.php [hunk 1]
```diff
@@ -47,6 +47,7 @@ public function date( $post_id, $taxonomy, $multiples = false, $data = array(),
 		}
 		
 		if ( isset($data[$taxonomy] ) ) {
+			date_default_timezone_set('America/Chicago');
```
Comment (willbarton) [inline inc/meta-box-callbacks.php]: Is it meaningful at all that the default is Chicago?

**Comment by willbarton** [comment]
:+1: Looks good to me, and tests pass in the test environment locally.

File: inc/meta-box-callbacks.php (hunk 1 above)
Comment (Scotchester) [inline inc/meta-box-callbacks.php]: Does seem like an odd choice...

Comment (kurtw) [inline inc/meta-box-callbacks.php]: I think I forgot to save a file before I committed...whoops. Thanks for catching it.

**Comment by willbarton** [comment]
selfie-0

**Comment by Scotchester** [comment]
Please update the version number in `cms-toolkit.php`. 1.3.1, since it's a patch-level update, I think. 1.4 if you think it's more significant than that. Sorry I didn't catch it before this got merged.

**Pull request merged**